pkg/location/type_dir.go

```diff
@@ -0,0 +1,33 @@
+package location
+
+import (
+	"fmt"
+	"io/fs"
+)
+
+type Dir struct {
+	Directory string
```

Comment:
with FileSystem abstraction, Directory alone is insufficient to identify a package.

We may need Dir that relies on local file system and one that is paired with FileSystem instance?